src/services/db/RepoService.ts

```diff
@@ -203,8 +203,25 @@ export default class RepoService extends GitHubService {
     return await super.getRepoState(sessionData)
   }
 
-  async getLatestCommitOfBranch(sessionData: any, branch: any): Promise<any> {
-    return await super.getLatestCommitOfBranch(sessionData, branch)
+  async getLatestCommitOfBranch(
+    sessionData: any,
+    branchName: any
+  ): Promise<any> {
```

Comment:
We can probably type the `sessionData` as UserWithSiteSessionData and the return to be `Promise<GitLocalDiskCommitData>`.

Reply:
oops, my bad, missed updating the types

Reply:
We can also specify `branchName` to be `string`

Reply:
The promise return type should be `Promise<GitHubCommitData>` if we incorporate[ this comment ](https://github.com/isomerpages/isomercms-backend/pull/891/files#r1290821301)

Reply:
done!